src/index.js

```diff
@@ -16,9 +16,19 @@ const root = document.getElementById(TicketEvolutionWindow.containerId)
 if (root) {
   // we don't want pointer events on text elements
   // also include css for tooltips
-  document.write(
-    '<style>#rootElement text{-webkit-touch-callout:none;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;pointer-events:none} .message-enter{opacity:.01}.message-enter-active{opacity:1;transition:all .3s ease-out}.message-exit{opacity:1}.message-exit-active{opacity:.01;transition:all .3s ease-out}</style>'
-  )
+  const css =
```

Comment:
@tevoaaron thanks for the heads up! 🥂  here's the recommended way to do this chicanery. 

and just for reference, document.write isn't deprecated, but frowned upon.

Reply:
@followbl - Chicanery indeed! I knew there was something about `document.write`.